### DISCUSSION TITLE
请问第二个命令remark找不到

### DISCUSSION THREAD

**Discussion by hezitai** [tidb-incubator/zh.md#3] [Q&A] [answered]
zsh: command not found: remark，是全局安装配置不对还是怎么回事

**Comment by wph95** [comment, marked as answer]
yeah, it's a bug, will fix it soon

**Reply by ithinco** [reply]
@WPH95 Got an error when running `remark -u preset-lint-zh xx.md`"Error: mdast-util-to-nlcst expected parser"

**Reply by wph95** [reply]
yeah, i think it's https://github.com/WPH95/retext-chinese/blob/main/src/parser.ts make a bug, working on fix it :)

**Reply by wph95** [reply]
0.2.0 version fixed

**Reply by hezitai** [reply]
谢谢！  问题已解决
